src/core/services/ripple/explode_ripple.js

```diff
@@ -0,0 +1,64 @@
+angular.module('material.core')
+    .directive('mdInkExplode', InkExplodeRippleDirective);
+
+
+/**
+ * @ngdoc directive
+ * @name mdInkExplode
+ * @module material.core
+ *
+ * @description
+ * The `md-ink-explode` directive allows you to specify a different container for the ripple to take place in when the
+ * element that this directive is on is clicked. This can be used for transitions to another state.
+ *
+ * @param {string} md-ink-explode A DOM selector for the container to be passed to QuerySelector()
+ * @param {number} md-ink-explode-dur The duration of the explode from start to finish
```

Comment:
When we provide this feature, then this should be IMO, `md-ink-explode-duration`.